rust/load/src/opentelemetry_config.rs

```diff
@@ -90,7 +90,25 @@ pub(crate) fn init_otel_tracing(service_name: &String, otel_endpoint: &String) {
     // global::set_tracer_provider(tracer_provider);
 
     // Prepare meter.
-    let client = reqwest::Client::new();
+    let mut headers = reqwest::header::HeaderMap::new();
+    headers.insert(
+        "x-honeycomb-dataset",
+        reqwest::header::HeaderValue::from_static("chroma-load"),
+    );
+    if let Ok(api_key) = std::env::var("OTEL_EXPORTER_OTLP_HEADERS") {
```

Comment:
nit: could we provide the dataset name in OTEL_EXPORTER_OTLP_HEADERS as config instead of special casing it here?